stratum/templates/podsecuritypolicy-stratum.yaml

```diff
@@ -0,0 +1,41 @@
+{{- if .Values.podsecuritypolicy.enabled }}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+    name: 1-stratum
+spec:
+  privileged: true
+  allowPrivilegeEscalation: true
+  volumes:
+    - 'configMap'
+    - 'secret'
+    - 'hostPath'
+  allowedHostPaths:
```

Comment:
@Yi-Tseng  any path needs to be removed in the current stratum version?

Reply:
Since we [disabled ONLP](https://github.com/stratum/stratum-helm-charts/blob/master/stratum/templates/configmap-startup.yaml#L17) support in this Helm chart
We can remove ONLP libraries (`libonlp-*.so` and `/lib/platform-config`) from here and the [daemon-set yaml](https://github.com/stratum/stratum-helm-charts/blob/master/stratum/templates/daemonset.yaml#L150-L158)
But we still need `/lib/modules/4.14.49-OpenNetworkLinux`, `/etc/onl`

Reply:
@frouzbeh can you help to change those as well?

Reply:
@hwchiu Yes, so just need to remove these paths?
- pathPrefix: "/lib/x86_64-linux-gnu/libonlp.so"
- pathPrefix: "/lib/platform-config"

Reply:
yes, and also remove related volume and volume mounts in the daemonset.yaml